docs/tools/sqlpackage.md

```diff
@@ -96,6 +96,8 @@ A SqlPackage.exe publish operation incrementally updates the schema of a target
 |**/AccessToken:**|**/at**|{string}| Specifies the token based authentication access token to use when connect to the target database. |
 |**/AzureKeyVaultAuthMethod:**|**/akv**|{Interactive&#124;ClientIdSecret}|Specifies what authentication method is used for accessing Azure KeyVault |
 |**/ClientId:**|**/cid**|{string}|Specifies the Client ID to be used in authenticating against Azure KeyVault, when necessary |
+|**/DeployScriptPath:**|**/dsp**|{string}|This is an optional path to write the deployment script to. For Azure deployment, if there were TSQL commands to create of modify the DB a master script will be written to the same path but with “Filename_Master.sql” as the output file name. |
+|**/DeployReportPath:**|**/drp**|{string}|This is an optional path to write the deployment report xml file to. |
```

Comment:
How about the following, again to be more inline with existing doc wording: Specifies an optional file path to output the deployment report xml file.

Reply:
PR updated